src/RazorSdk/SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.csproj

```diff
@@ -30,6 +31,8 @@
 
   <Target Name="GetDependencyTargetPaths">
     <ItemGroup>
+      <TargetPathWithTargetPlatformMoniker Include="$(PkgMicrosoft_AspNetCore_Razor_Language)\lib\netstandard2.0\Microsoft.AspNetCore.Razor.Language.dll" IncludeRuntimeDependency="false" />
```

Comment:
To confirm, we don't have compiler binaries in this SourceGenerators dir, yes?

Reply:
Now it does. 😄

Reply:
Sorry, I meant Microsoft.CodeAnalysis.CSharp etc

Reply:
Yep -- I caught your drift. The current contents of the source-generators directory is above which is just the stuff we include in `<Analyzers>` and the EnC config file.